docs/src/dev/provider/gremlin-semantics.asciidoc

```diff
@@ -1517,7 +1517,7 @@ link:https://tinkerpop.apache.org/docs/x.y.z/reference/#rTrim-step[reference]
 *Arguments:*
 
 * `separator` - The string character(s) used as delimiter to split the input string. Nullable, a null separator will split on
-whitespaces.
+whitespaces. An empty separator will split on each character.
```

Comment:
Minor nit: I think it should probably be "empty string separator"

Reply:
Changed to 'empty string separator' for this file and also the-traversal.asciidoc.